ui/Gruntfile.js

```diff
@@ -25,7 +25,7 @@ var DEFAULT_PROXY_PROFILE = {
       context: '/pnc-rest',
       host: 'localhost',
       port: 8080,
-      ws: true
+      wss: true
```

Comment:
Hi Martin, @matedo1 
I am not sure about this part because I haven't found out how the proxy-profiles.json works with pnc-WebUI, if this doesn't look right please let me know.

Reply:
Hi @DnsZhou check documentation, I think this should be the best place to investigate: https://github.com/mrobert44/grunt-connect-proxy-updated/blob/9f1044d48a8edf3c34c81072a9b6416ea573c359/README.md

Reply:
Great, thanks. I was not sure whether it is consumed by which component.